scripts/release.sh

```diff
@@ -39,4 +39,15 @@ gh release upload "$RELEASE_TAG" live_compositor_with_web_renderer_linux_x86_64.
 gh release upload "$RELEASE_TAG" live_compositor_with_web_renderer_darwin_x86_64.tar.gz
 gh release upload "$RELEASE_TAG" live_compositor_with_web_renderer_darwin_aarch64.tar.gz
 
+IMAGE_NAME="ghcr.io/software-mansion/live-compositor"
+COMMIT_HASH="$( git rev-parse HEAD )"
```

Comment:
it needs to be an env here, this script will probably be called on a different commit